### PR TITLE
Getting CLI docs ready for RTM

### PR DESCRIPTION
**Pull request by thesmithee**
One thing that I didn't add was the explanation about the notice on first run. I don't know if this is needed at all. What do you think @mairaw and @richlander?

### CODE REVIEW

**Comment by mairaw** [comment]
what about the RC2 note on the app types topic?
https://github.com/dotnet/core-docs/blob/master/docs/core-concepts/app-types.md

**Comment by blackdwarf** [comment]
Ugh, good catch @mairaw. Will fix presently. The behavior hasn't changed from RC2 so...though this is a good question, it should state "tools Preview 2" or similar, since this is a tooling issue. @richlander any comments?

**Comment by mairaw** [comment]
Thanks for making the changes for me, @blackdwarf. LGTM

**Comment by blackdwarf** [comment]
OK, don't merge until RTM/Preview 2 ships.

**Comment by mairaw** [comment]
@blackdwarf the merges are not going live to dotnet.github.io this week. Feel free to merge away! :rocket:

File: docs/core-concepts/core-sdk/cli/dotnet-new.md [hunk 1]
```diff
@@ -25,18 +25,22 @@ Language of the project. Defaults to `C#`. `csharp` (`fsharp`) or `cs` (`fs`) ar
...
 
 `dotnet new`
     
-Drops a sample C# project in the current directory.
+Drops a C# project in the current directory.
 
 `dotnet new --lang f#`
     
-Drops a sample F# project in the current directory.
+Drops a F# project in the current directory.
```
Comment (svick) [inline docs/core-concepts/core-sdk/cli/dotnet-new.md]: Should this be "an F# project"?

**Comment by blackdwarf** [comment]
@mairaw merging away! 😄

**Pull request merged**